calculator.go

```diff
@@ -16,3 +18,12 @@ func Subtract(a, b float64) float64 {
 func Multiply(a, b float64) float64 {
 	return a * b
 }
+
+// Divide takes two numbers and returns the result by dividing the first by the second
+func Divide(a, b float64) (float64, error) {
```

Comment:
Great! Now, for extra credit, could we take _any_ number of parameters to Divide, and return the result of the entire sum? In fact, we could do that for Add, Subtract, and Multiply too, couldn't we? Or rather, they must take _at least two_ parameters, for the operation to make sense, but then we'd like to be able optionally supply N extra parameters. How could we do that?

Reply:
> Great! Now, for extra credit, could we take _any_ number of parameters to Divide, and return the result of the entire sum? In fact, we could do that for Add, Subtract, and Multiply too, couldn't we? Or rather, they must take _at least two_ parameters, for the operation to make sense, but then we'd like to be able optionally supply N extra parameters. How could we do that?

Are we still using floating point parameters for this?

Reply:
Well, that would be handy, but it doesn't really matter. You can take a variable number of `int` parameters if you prefer!